site/_docs/tutorial.md

```diff
@@ -58,7 +58,7 @@ You need Java (version 8, 9 or 10) and git.
 {% highlight bash %}
 $ git clone https://github.com/apache/calcite.git
 $ cd calcite
-$ ./mvnw install -DskipTests -Dcheckstyle.skip=true
+$ ./mvnw clean install -DskipTests -Dcheckstyle.skip=true
 $ cd example/csv
```

Comment:
What if we do not have `clean` as options ?

Reply:
You can see the code in `sqlline`.
If we do not use `clean`, it will use old `taget/fullclasspath.txt`(this file includes the dependencies the project will use), it will throws some errors while executing `sqlline` in some cases(for example: the Calcite project add a new adapter).

Reply:
Then it makes sense to document that.
Or update the logic that creates `taget/fullclasspath.txt` so the file is updated.

Reply:
@vlsi @danny0405 @chunweilei The `taget/fullclasspath.txt` file is created while executing `sqlline`.
If the file exists we can notice user, or create a new file forcibly. I think we can create a new `taget/fullclasspath.txt` file forcibly while executing `sqlline`, What do you think?

Reply:
Note: not everybody uses sqlline. Adding "clean" for every build would severely impact build duration, so it looks like an overkill.

As for me, the proper solution would be https://issues.apache.org/jira/browse/CALCITE-2905

Reply:
@vlsi I agree with you, it's not necessary to execute `clean` for  every build or update the `taget/fullclasspath.txt` for every time while executing `sqlline`.